provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -71,6 +71,9 @@ data:
         "service-vlan": {{ config.net_config.service_vlan|json }},
         "encap-type": {{ config.node_config.encap_type|json }},
         "aci-infra-vlan": {{ config.net_config.infra_vlan|json }},
+        {% if config.net_config.interface_mtu %}
+        "interface-mtu": {{ config.net_config.interface_mtu }},
```

Comment:
Does this need the '|json" filter as is being used for other configs?

Reply:
Good catch, I will fix that.